anymail/backends/scaleway.py

```diff
@@ -0,0 +1,163 @@
+                message_id=message_id, status=status_map.get(status, "unknown")
+            )
+            if recipient:
+                statuses[recipient] = anymail_status
+        return statuses
+
+
+class ScalewayPayload(RequestsPayload):
+    def __init__(self, message, defaults, backend, *args, **kwargs):
+        self.project_id = backend.project_id
+        http_headers = kwargs.pop("headers", {})
+        http_headers["X-Auth-Token"] = backend.api_key
+        http_headers["Content-Type"] = "application/json"
+        super().__init__(
+            message, defaults, backend, headers=http_headers, *args, **kwargs
+        )
+
+    def get_api_endpoint(self):
+        return "emails"
+
+    def init_payload(self):
+        self.data = {
+            "to": [],
+            "cc": [],
+            "bcc": [],
+            "additional_headers": [],
+            "attachments": [],
+        }
+        if self.project_id:
+            self.data["project_id"] = self.project_id
+
+    def set_from_email(self, email):
+        self.data["from"] = {
+            "email": email.addr_spec,
+            "name": email.display_name or None,
+        }
+
+    def set_recipients(self, recipient_type, emails):
+        if emails:
+            self.data[recipient_type] = [
+                {"email": email.addr_spec, "name": email.display_name or None}
+                for email in emails
+            ]
+
+    def set_subject(self, subject):
+        if subject:
+            self.data["subject"] = subject
+
+    def set_text_body(self, body):
+        if body:
+            self.data["text"] = body
+
+    def set_html_body(self, html):
+        if html:
+            self.data["html"] = html
+
+    def add_attachment(self, attachment):
+        self.data.setdefault("attachments", []).append(
+            {
+                "name": attachment.name,
+                "type": attachment.mimetype,
+                "content": attachment.b64content,
+            }
+        )
+
+    def set_extra_headers(self, headers):
+        self.data.setdefault("additional_headers", []).extend(
+            [{"key": key, "value": value} for key, value in headers.items()]
+        )
+
+    def set_reply_to(self, emails):
+        if emails:
+            reply_to_string = ", ".join([str(email) for email in emails])
+            self.data.setdefault("additional_headers", []).append(
+                {"key": "Reply-To", "value": reply_to_string}
+            )
+
+    def serialize_data(self):
+        # Clean up empty lists
+        if not self.data.get("to"):
+            del self.data["to"]
+        if not self.data.get("cc"):
+            del self.data["cc"]
+        if not self.data.get("bcc"):
+            del self.data["bcc"]
+        if not self.data.get("attachments"):
+            del self.data["attachments"]
+        if not self.data.get("additional_headers"):
+            del self.data["additional_headers"]
+        return self.serialize_json(self.data)
```

Comment:
```suggestion
    def serialize_data(self):
        return self.serialize_json(self.data)
```

(If you skip creating the empty lists in init_payload.)